UDx-container/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Isn't it better to use `/usr/bin/env` to find the bash executable?  Wouldn't that make it compatible in more environments.